packages/financial-templates-lib/src/price-feed/DefiPulseTotalPriceFeed.js

```diff
@@ -0,0 +1,163 @@
+const { PriceFeedInterface } = require("./PriceFeedInterface");
+const { parseFixed } = require("@ethersproject/bignumber");
+const assert = require("assert");
+
+// An implementation of PriceFeedInterface that uses DefiPulse Data api to retrieve prices.
+class DefiPulseTotalPriceFeed extends PriceFeedInterface {
+  /**
+   * @notice Constructs the DefiPulseTVL_ALLPriceFeed.
+   * @param {Object} logger Winston module used to send logs.
+   * @param {Object} web3 Provider from truffle instance to connect to Ethereum network.
+   * @param {String} apiKey DeFiPulse Data API key. Note: these API keys are rate-limited.
+   * @param {Integer} lookback How far in the past the historical prices will be available using getHistoricalPrice.
+   * @param {Object} networker Used to send the API requests. 
+   * @param {Function} getTime Returns the current time.
+   * @param {Integer} minTimeBetweenUpdates Min number of seconds between updates. If update() is called again before
+   *      this number of seconds has passed, it will be a no-op.
+   */
+  constructor(
+    logger,
+    web3,
+    apiKey,
+    lookback,
+    networker,
+    getTime,
+    minTimeBetweenUpdates,
+    decimals = 18,
+  ) {
+    super();
+    this.logger = logger;
+    this.web3 = web3;
+
+    this.apiKey = apiKey;
+    assert(apiKey, "requires apiKey for Defi Pulse");
+
+    this.lookback = lookback;
+    this.networker = networker;
+    this.getTime = getTime;
+    this.minTimeBetweenUpdates = minTimeBetweenUpdates;
+
+    this.toBN = this.web3.utils.toBN;
+
+    this.historicalPrices = [{timestamp:0, value:0}];
+
+    this.convertDecimals = number => {
+      // Converts price result to wei
+      // returns price conversion to correct decimals as a big number
+      return this.toBN(parseFixed(number.toString(), decimals).toString());
+    };
+
+    this.removeValuesOlderThanLookback = () => {
+
+        const currentTime = this.getTime();
+
+        this.historicalPrices = this.historicalPrices.filter(priceObj => priceObj.timestamp > (currentTime - lookback) );
+
+    }
+
+
```

Comment:
this file still needs to be linted

Reply:
I tried running yarn and then yarn lint-fix on the repo but got "lint-fix" not found. a) what linter should I be using? b) is there a config file for the linter somewhere in the repo?